.env

```
@@ -2,31 +2,49 @@
 
 # User Configurable Variables
 ## Webapp URL
-CHAOSGENIUS_WEBAPP_URL=http://localhost:8080/  # URL of the Chaos Genius deployment. Usually, this will be http://<ip-address-or-hostname>:8080/ (http://localhost:8080/ in local installations).
+# URL of the Chaos Genius deployment. Usually, this will be http://<ip-address-or-hostname>:8080/ (http://localhost:8080/ in local installations).
+CHAOSGENIUS_WEBAPP_URL=http://localhost:8080/  
```

There are trailing spaces here. I think docker-compose 1.25 will include the spaces too if they are present.
```suggestion
CHAOSGENIUS_WEBAPP_URL=http://localhost:8080/
```